research-hub-web/src/app/components/subhubs/subhubs.component.html

```diff
@@ -0,0 +1,75 @@
+<mat-card class="parent-results" *ngIf="parentSubHubs.length > 0">
+    <!-- <pre><code>{{ parentSubHubs | json}}</code></pre> -->
```

Comment:
Commented out code can be removed.

Reply:
Removed (hope you don't mind @warrickct ) ✌️